### ISSUE TITLE
autologin stopped working

### ISSUE THREAD

**Issue by oberon2007** [manjaro/manjaro-tools#186]
Recently autologin with Cinnamon-ISOs was working fine.
Just now I built a Cinnamon-minimal and no autologin.
**/etc/lightdm/lightdm.conf** is looking fine, I think (commented lines removed for readability)
```
[LightDM]
run-directory=/run/lightdm

[Seat:*]
pam-autologin-service=lightdm-autologin
autologin-user=manjaro
autologin-user-timeout=0
```

**Comment by udeved** [comment]
Whats the greeter-session and user-session in lightdm.conf?

**Comment by oberon2007** [comment]
``#greeter-session=example-gtk-gnome``
and
``#user-session=default``

**Comment by oberon2007** [comment]
commented out!

**Comment by udeved** [comment]
Hmm, I am currently buiding xfce-minimal, and auto detect appears to work.

~~~
==> Configuring [xfce-image]
  -> Setting plymouth manjaro-elegant ....
  -> Configuring Displaymanager ...
 --> Loading [/usr/share/manjaro-tools/desktop.map] ...
  -> Trying to detect desktop environment ...
  -> Detected: xfce
  -> Configured: lightdm
 --> Configuring [systemd] ....
  -> Setting bluetooth ...
  -> Setting cronie ...
  -> Setting ModemManager ...
  -> Setting NetworkManager ...
  -> Setting org.cups.cupsd ...
  -> Setting tlp ...
  -> Setting tlp-sleep ...
 --> Done configuring [systemd]
~~~

lightdm.conf in xfce chroot is correctly set....
~~~
user-session=xfce
~~~

greeter-session is only set if the greeter is else but gtk.

**Comment by oberon2007** [comment]
where's this buildlog to be found?

**Comment by udeved** [comment]
buildlog is a feature to be implemented in 0.11

**Comment by oberon2007** [comment]
So this is just your terminal-output?

**Comment by udeved** [comment]
right

**Comment by udeved** [comment]
Please rebuild the tools, autologin appears to work.

**Comment by oberon2007** [comment]
Yes, working now :) Thank you!

**Comment by philmmanjaro** [comment]
What happend? I had that fixed already with: 3c77b381dcf66a620eaaa7d0e901e7d05bca0c32

**Comment by philmmanjaro** [comment]
@udeved: please **git pull** before you push some. Sometimes I fix some bugs automatically ;)
